nin/dasBoot/Node.js

```diff
@@ -0,0 +1,22 @@
+(function(NIN) {
+  'use strict';
+
+  class Node {
+    constructor(id, options) {
+      this.id = id;
+      this.inputs = options.inputs || {};
+      this.outputs = options.outputs || {};
+    }
+
+    resize() {
```

Comment:
Is there such a thing as forcing a subclass to implement functions from it's parent?
This would be a good fit for abstract / virtual.